compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -903,9 +903,8 @@ impl CrateInfo {
         // If global LTO is enabled then almost everything (*) is glued into a single object file,
         // so this logic is not necessary and can cause issues on some targets (due to weak lang
         // item symbols being "privatized" to that object file), so we disable it.
-        // (*) Native libs, and `#[compiler_builtins]` and `#[no_builtins]` crates are not glued,
-        // and we assume that they cannot define weak lang items. This is not currently enforced
-        // by the compiler, but that's ok because all this stuff is unstable anyway.
+        // (*) Native libs are not glued, and we assume that they cannot define weak lang items.
+        // This is not currently enforced by the compiler, but that's ok because all this stuff is unstable anyway.
```

Comment:
(This sentence was also about compiler_builtins/no_builtins and can be removed.)